lib/common/options.c

```diff
@@ -956,6 +1234,35 @@ pcmk__output_cluster_options(pcmk__output_t *out, const char *name,
                         cluster_options, all);
 }
 
+/*!
+ * \internal
+ * \brief Output fence device common instance attribute metadata as OCF-like XML
```

Comment:
TODO: Update all these doxygen blocks and descriptions to use "parameters" rather than "instance attributes". Might as well be proactive based on https://github.com/ClusterLabs/pacemaker/pull/3391#discussion_r1537703410.